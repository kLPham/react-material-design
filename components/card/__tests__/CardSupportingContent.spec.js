import React from 'react';
import { shallow } from 'enzyme';
import CardSupportingContent from '../cardSupportingContent';

describe('<CardSupportingContent />', () => {
    it('should render', () => {
        const cardSupportingContent = shallow(
            <CardSupportingContent>Text</CardSupportingContent>,
    );
        expect(cardSupportingContent).toMatchSnapshot();
    });
});
