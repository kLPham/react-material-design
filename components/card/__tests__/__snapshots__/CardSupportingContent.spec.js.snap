// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardSupportingContent /> should render 1`] = `
<section
  className="mdc-card__supporting-text"
>
  Text
</section>
`;
